pkgs/tools/inputmethods/input-remapper/default.nix

```diff
@@ -59,9 +51,6 @@ buildPythonApplication {
   strictDeps = false;
 
   prePatch = ''
-    # set revision for --version output
-    echo "COMMIT_HASH = '${input_remapper_src_rev}'" > inputremapper/commit_hash.py
-
     # fix FHS paths
     substituteInPlace inputremapper/configs/data.py \
       --replace "/usr/share/input-remapper"  "$out/usr/share/input-remapper"
```

Comment:
Also all the () in the line below can be removed.

Reply:
Done